src/main/resources/default/taglib/t/videoAcademyBtn.html.pasta

```diff
@@ -0,0 +1,25 @@
+<i:arg type="String" name="code"/>
+<i:arg type="String" name="class" default=""/>
+<i:arg type="String" name="labelKey" default="OnboardingController.videoAcademyBtnLabel"/>
+<i:arg type="String" name="label" default="@i18n(labelKey)"/>
+
+<i:pragma name="description">
+    Provides a button linking to the onboarding video with the given code.
+    If no matching video is found, no button will be rendered. If the video is recommended to the user,
+    the button is rendered as "info" button, otherwise a gray outline button is shown.
+</i:pragma>
+
+<i:local name="engine" value="Injector.context().getPart(sirius.biz.tycho.academy.OnboardingEngine.class)" />
```

Comment:
Can't we use @<!-- -->part here?

Reply:
Nope, @part fails if the dependency is null - However, the onboarding framework might be simply disabled - in this case we rather want to do nothing instead of crashing the template :-P

Reply:
Thank for explaining!